.github/workflows/test-template.yml

```
@@ -102,7 +102,7 @@ jobs:
   # CLIENT (functional and legacy tests without types test)
   #
   client:
-    name: Client (functional and legacy tests without types test)
+    name: Client func/legacy notypes
```

Maybe add a "-" to make it clear that "notypes" pertains to legacy?
```suggestion
    name: Client func/legacy-notypes
```